packages/atomic/src/components/common/suggestions/suggestions-common.ts

```diff
@@ -1,204 +1,25 @@
-import type {VNode} from '@stencil/core';
-import type {HTMLStencilElement} from '@stencil/core/internal';
-import type {LitElement} from 'lit';
-import {closest} from '../../../utils/dom-utils';
-import {buildCustomEvent} from '../../../utils/event-utils';
-import type {AnyBindings} from '../interface/bindings';
-
-/**
- * Element which will be rendered in the list of suggestions.
- */
-export interface SearchBoxSuggestionElement {
-  /**
-   * Stable identity which enables Stencil to reuse DOM elements for better performance.
-   * The best way to pick a key is to use a string that uniquely identifies that list item among its siblings (often your data will already have IDs).
-   */
-  key: string;
-  /**
-   * Rendered content of the element.
-   *
-   * @remarks
-   * The `VNode` type will be deprecated in v4 as we are detaching from Stencil.
-   */
-  content: Element | VNode;
-  /**
-   * Hook called when the selection is selected.
-   * @param e DOM event.
-   */
-  onSelect?(e: Event): void;
-  /**
-   * The query associated with the suggestion which will replace the query in the search box if the suggestion is selected.
-   */
-  query?: string;
-  /**
-   * For improved accessibility, provide this property with additional information.
-   * https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label
-   */
-  ariaLabel?: string;
-  /**
-   * Adds a specific shadow part attribute that can be selected with the CSS ::part pseudo-element.
-   * https://developer.mozilla.org/en-US/docs/Web/CSS/::part
-   */
-  part?: string;
-  /**
-   * Hide the suggestion if it's the last in the list.
-   */
-  hideIfLast?: boolean;
-}
-
 /**
- * List of suggestions that will be displayed along other lists (e.g recent queries) when the search box's input is selected.
+ * @fileoverview Re-exports all suggestion-related functionality for backward compatibility.
```

Comment:
Can't we just update the paths of these exports in the index.ts and get rid of this file (and its spec file) entirely? 

We define the @coveo/atomic export entry points explicitly in the package.json, so as long as we don't change the export names, this should be non-breaking I think.